docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,69 +1,269 @@
-[[get-started-docker]]
-== Running the {stack} on Docker
+[[get-started-stack-docker]]
+== Running the {stack} ("ELK") on Docker
+++++
+<titleabbrev>Running the {stack} on Docker</titleabbrev>
+++++
 
 The Elastic Docker registry contains Docker images for all the products in
 the {stack}: https://www.docker.elastic.co/.
 
-[float]
-[[run-stack-docker]]
-=== Run with Docker Compose
+You can <<run-docker-secure,start the {stack} in Docker>> with security enabled
+and configured by default. This option is great for quickly getting started with
+{es} and {kib} for testing and development.
+
+You can also <<get-started-docker-tls,start the {stack} with Docker Compose>> to
+create a secured, multi-node cluster with a connected {kib} instance. This
+option requires more manual steps, but results in a more robust cluster.
+
+[[run-docker-secure]]
+[discrete]
+=== Start {es} and {kib} in Docker
+Starting in {es} 8.0, security is enabled by default. With security enabled,
+{stack} {security-features} require TLS encryption for the transport networking
+layer, or your cluster will fail to start.
+
+If you're starting a single-node {es} cluster in a Docker container, security
+will be automatically enabled and configured for you. When you start {es} for
+the first time, the following security configuration occurs automatically:
+
+* {ref}/configuring-stack-security.html#stack-security-certificates[Certificates and keys] are generated
+for the transport and HTTP layers.
+* The Transport Layer Security (TLS) configuration settings are written to
+`elasticsearch.yml`.
+* A password is generated for the `elastic` user.
+* An enrollment token is generated for {kib}.
+
+You can then {kibana-ref}/docker.html[start {kib}] and enter the enrollment
+token, which is valid for 30 minutes. This token automatically applies the
+security settings from your {es} cluster, authenticates to {es} with the
+`kibana_system` user, and writes the security configuration to `kibana.yml`.
+
+[discrete]
+==== Prerequisites
+. Install the appropriate https://docs.docker.com/get-docker/[Docker application]
+for your operating system.
++
+[NOTE]
+====
+Make sure that Docker is allotted at least 4GiB of memory. In Docker Desktop,
```

Comment:
Is that still valid ? I run my docker containers with less with no issue

Reply:
It's definitely valid for running a multi-node cluster with Docker Compose. With less memory than this, one node in my cluster continually crashed on macOS. However, this setting might not apply to the single-node cluster. If that's the case, I'll move this note to the section for using Docker Compose.

Reply:
For a single-node cluster, I can run both the Elasticsearch container and the Kibana container with only 1.5 GB of memory, provided that no other containers are running. If I have another container running (like a local doc build) and try to start both Elasticsearch and Kibana, Docker stops one of those containers to regain memory.